ndcube/asdf/resources/schemas/ExtraCoords-0.1.0.yaml

```diff
@@ -0,0 +1,34 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/ExtraCoords-0.1.0"
+
+title:
+  Represents the ndcube ExtraCoords object
+
+description:
+  Represents the ndcube ExtraCoords object
+
+type: object
+properties:
+  wcs:
+    tag: "tag:stsci.edu:gwcs/wcs-1.*"
+  mapping:
+    type: array
+  lookup_tables:
```

Comment:
I am unsure about the structure of the `lookup_tables` under the properties validator in the `ExtraCoords` schema and would need to test it with more examples. (I'll update it soon)